astropy/analytic_functions/tests/test_blackbody.py

```diff
@@ -32,7 +32,7 @@ def test_blackbody_scipy():
 
     """
     flux_unit = u.Watt / (u.m ** 2 * u.um)
-    wave = np.logspace(0, 8, 1e6) * u.AA
+    wave = np.logspace(0, 8, 100000) * u.AA
```

Comment:
I just didn't see a reason to make the arrays that large, as testing takes long as it is, and this is starting to become non-trivial (at least, on my relatively slow laptop, 1 million elements takes 440 ms, while 100k takes 70 ms).